### PR TITLE
jsapi-3.21

### PR DESCRIPTION
**Pull request by jwasilgeo**
After this is accepted and merged, I will do the following (likely not until week of 7/17):

- prep the repo for a new v1.x release (package.json)
- publish a new v1.x release to github and npm
- update the this branch's separate gh-pages repo so that the public docs site has all the latest and greatest

### CODE REVIEW

File: src/core/esriLoader.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
          * Loads the ArcGIS API for JavaScript.
          *
          * @param {Object=} options Send a list of options of how to load the ArcGIS API for JavaScript.
-         *  Defaults to `{url: '//js.arcgis.com/3.20compact'}`
```
Comment (andygup) [inline src/core/esriLoader.js]: I recommend explicitly adding `https` to the `url` property

File: src/core/esriLoader.js [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
+         *  Defaults to `{url: '//js.arcgis.com/3.21compact'}`
```
Comment (andygup) [inline src/core/esriLoader.js]: Same comment as above with respect to adding `https`

**Comment by jwasilgeo** [comment]
Thank you, @andygup. How about now?

**Comment by andygup** [comment]
Also looks good, thanks!

**Pull request merged**